apps/api/v2/src/modules/billing/services/billing.service.ts

```diff
@@ -43,12 +49,9 @@ export class BillingService implements OnModuleDestroy {
 
   async createSubscriptionForTeam(teamId: number, plan: PlatformPlan) {
```

Comment:
in follow up pr, let's refactor this in 2 different functions doing only one thing

- createTeamBilling, which just returns the customerId or the whole platforBillingEntity
- redirectToSubscribeCheckout, which returns the url